docs/self-hosting-docker.md

```
@@ -15,41 +15,76 @@ For this guide, we assume that before you start, you have a dedicated docker fol
+   ```bash
+   openssl rand -base64 48   # JWT_SECRET
+   openssl rand -base64 24   # DB_PASSWORD
+   openssl rand -hex 32      # ENCRYPTION_KEY  (must be 64 hex characters)
+   openssl rand -base64 48   # SESSION_SECRET
+   openssl rand -base64 24   # MINIO_SECRET_KEY
+   ```
+
+   Then create the file with `nano .env` and paste in the following, replacing
+   each placeholder with the matching output above:
+
+   ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)